modules/imaging_uploader/jsx/UploadForm.js

```diff
@@ -143,6 +142,31 @@ class UploadForm extends Component {
           hasError.visitLabel = true;
         }
 
+        this.setState({errorMessage, hasError});
+        return;
+      }
+      if (!fileName.match(properExt)) {
+        swal({
+          title: 'Invalid extension for the uploaded file!',
+          text: 'Filename extension does not match .zip, .tgz or .tar.gz ',
```

Comment:
It looks like this line is not being executed when the file extension is incorrect.

Reply:
@kchatpar you left this comment and then approved the review. did the swal end up working?

Reply:
@zaliqarosli `npm run compile` was not run for the first test so the comment above was due to the fact that the JS files were not compiled. He tested again after compiling the JS files and now it works.